_scss/wallscreens/_modal.scss

```diff
@@ -101,4 +102,17 @@
     background-color: var(--color-primary-dark);
     color: white;
   }
+
+  .debug-info {
+    position: absolute;
+    bottom: 1em;
+    left: 1em;
+    font-size: 1rem;
+    line-height: 1;
+    color: $su-color-stone;
+  }
+
+  .build-date::before {
+    content: "• ";
```

Comment:
@thatbudakguy this is a minor thing, but I wonder if we should use a separator with less visual weight (maybe a dash) to help avoid drawing non-staff attention to the debug info.

Reply:
makes sense; changed to a dash